### ISSUE TITLE
Unable to click on the data points in LineChartView when using Talkback

### ISSUE THREAD

**Issue by dashengz** [diogobernardino/WilliamChart#121]
I'm not sure if I had overlooked the function or not, but right now the points in the LineChartView is not focusable, and if I use Talkback, it's not be possible for me to click on the data points.

Is it possible to make the data points focusable and set their contentDescription in LineChartView?

**Comment by diogobernardino** [comment]
I believe for that they needed to be Views, which is not the case.

**Comment by dashengz** [comment]
Got it, Thanks! I created transparent views for each point and solved the problem.
